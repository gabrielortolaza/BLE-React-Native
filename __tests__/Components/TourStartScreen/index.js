import React from "react";
import renderer from "react-test-renderer";

const TourStartScreen = require("../../../src/Components/TourStartScreen").default;

describe("TourStartScreen", () => {
  it("Render TourStartScreen", () => {
    expect(renderer.create(<TourStartScreen />).toJSON()).toMatchSnapshot();
  });
});
