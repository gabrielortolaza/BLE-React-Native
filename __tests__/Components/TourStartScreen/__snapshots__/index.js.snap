// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`TourStartScreen Render TourStartScreen 1`] = `
<View
  style={
    Array [
      Object {
        "backgroundColor": "#fff",
        "flex": 1,
        "height": 1334,
      },
      Object {
        "backgroundColor": "rgb(66, 162, 194)",
        "flex": 1,
      },
    ]
  }
  testID="TourStartScreen"
>
  <View
    style={
      Object {
        "height": 844,
        "position": "absolute",
        "right": 0,
        "top": 10,
        "width": 600,
      }
    }
  >
    <Image
      resizeMode="cover"
      source={
        Object {
          "testUri": "../../../src/Assets/Images/tourStartPumppy.png",
        }
      }
      style={
        Object {
          "flex": 1,
          "height": null,
          "resizeMode": "cover",
          "width": null,
        }
      }
    />
  </View>
  <RCTScrollView
    automaticallyAdjustContentInsets={false}
    contentContainerStyle={
      Array [
        Object {
          "padding": undefined,
        },
        undefined,
      ]
    }
    contentInset={
      Object {
        "bottom": 0,
      }
    }
    enableAutomaticScroll={true}
    enableOnAndroid={false}
    enableResetScrollToCoords={true}
    extraHeight={75}
    extraScrollHeight={0}
    getScrollResponder={[Function]}
    handleOnScroll={[Function]}
    keyboardDismissMode="interactive"
    keyboardOpeningTime={250}
    keyboardShouldPersistTaps="handled"
    keyboardSpace={0}
    onScroll={[Function]}
    resetKeyboardSpace={[Function]}
    resetScrollToCoords={
      Object {
        "x": 0,
        "y": 0,
      }
    }
    scrollEventThrottle={1}
    scrollForExtraHeightOnAndroid={[Function]}
    scrollIntoView={[Function]}
    scrollToEnd={[Function]}
    scrollToFocusedInput={[Function]}
    scrollToPosition={[Function]}
    showsVerticalScrollIndicator={true}
    style={
      Array [
        Object {
          "backgroundColor": "transparent",
          "flex": 1,
        },
        Object {
          "flex": 1,
          "paddingBottom": 80,
          "paddingLeft": 76,
          "paddingRight": 54,
          "paddingTop": 220,
        },
      ]
    }
    testID="welcome-test"
    update={[Function]}
    viewIsInsideTabBar={false}
  >
    <View>
      <Text
        style={
          Array [
            Object {
              "fontFamily": "Graphik-Regular",
              "letterSpacing": 0.75,
              "textAlign": "left",
            },
            undefined,
            undefined,
            undefined,
            undefined,
            undefined,
            undefined,
            undefined,
            undefined,
            undefined,
            undefined,
            undefined,
            undefined,
            undefined,
            undefined,
            undefined,
            undefined,
            undefined,
            undefined,
            undefined,
            undefined,
            undefined,
            undefined,
            undefined,
            undefined,
            undefined,
            undefined,
            undefined,
            undefined,
            undefined,
            undefined,
            undefined,
            Object {
              "color": "white",
              "fontFamily": "Graphik-Semibold",
              "fontSize": 60,
              "lineHeight": 72,
            },
          ]
        }
      >
        Welcome to
        

        Pumpables!
      </Text>
      <Text
        style={
          Array [
            Object {
              "fontFamily": "Graphik-Regular",
              "letterSpacing": 0.75,
              "textAlign": "left",
            },
            undefined,
            undefined,
            undefined,
            undefined,
            undefined,
            undefined,
            undefined,
            undefined,
            undefined,
            undefined,
            undefined,
            undefined,
            undefined,
            undefined,
            undefined,
            undefined,
            undefined,
            undefined,
            undefined,
            undefined,
            undefined,
            undefined,
            undefined,
            undefined,
            undefined,
            undefined,
            undefined,
            undefined,
            undefined,
            undefined,
            undefined,
            Object {
              "color": "white",
              "fontSize": 40,
              "lineHeight": 60,
              "marginBottom": 60,
              "marginTop": 100,
            },
          ]
        }
      >
        Get started with the Pumpables app to operate your SuperGenie, track your pumping and feeding sessions, your milk stash and more.
      </Text>
      <View
        accessible={true}
        focusable={true}
        isTVSelectable={true}
        onClick={[Function]}
        onResponderGrant={[Function]}
        onResponderMove={[Function]}
        onResponderRelease={[Function]}
        onResponderTerminate={[Function]}
        onResponderTerminationRequest={[Function]}
        onStartShouldSetResponder={[Function]}
        style={
          Object {
            "alignItems": "center",
            "backgroundColor": "rgb(255, 255, 255)",
            "borderBottomWidth": null,
            "borderColor": "#007aff",
            "borderLeftWidth": null,
            "borderRadius": 25,
            "borderRightWidth": null,
            "borderTopWidth": null,
            "borderWidth": undefined,
            "elevation": 0,
            "flexDirection": "row",
            "height": 50,
            "justifyContent": "center",
            "marginBottom": 10,
            "minWidth": 100,
            "opacity": 1,
            "paddingBottom": 6,
            "paddingHorizontal": 15,
            "paddingTop": 6,
            "paddingVertical": 10,
            "shadowColor": undefined,
            "shadowOffset": undefined,
            "shadowOpacity": undefined,
            "shadowRadius": undefined,
            "width": 320,
          }
        }
      >
        <Text
          style={
            Array [
              Object {
                "fontFamily": "Graphik-Regular",
                "letterSpacing": 0.75,
                "textAlign": "left",
              },
              undefined,
              undefined,
              undefined,
              undefined,
              undefined,
              undefined,
              Object {
                "color": "rgb(66, 162, 194)",
              },
              undefined,
              undefined,
              undefined,
              undefined,
              undefined,
              undefined,
              undefined,
              undefined,
              Object {
                "fontSize": 18,
              },
              undefined,
              undefined,
              undefined,
              undefined,
              undefined,
              undefined,
              undefined,
              undefined,
              undefined,
              undefined,
              undefined,
              undefined,
              undefined,
              undefined,
              undefined,
              undefined,
            ]
          }
        >
          Start Now
        </Text>
      </View>
      <View
        accessible={true}
        focusable={true}
        isTVSelectable={true}
        onClick={[Function]}
        onResponderGrant={[Function]}
        onResponderMove={[Function]}
        onResponderRelease={[Function]}
        onResponderTerminate={[Function]}
        onResponderTerminationRequest={[Function]}
        onStartShouldSetResponder={[Function]}
        style={
          Object {
            "alignItems": "center",
            "backgroundColor": "transparent",
            "borderBottomWidth": null,
            "borderColor": "#007aff",
            "borderLeftWidth": null,
            "borderRadius": 25,
            "borderRightWidth": null,
            "borderTopWidth": null,
            "borderWidth": undefined,
            "elevation": 0,
            "flexDirection": "row",
            "height": 50,
            "justifyContent": "flex-start",
            "minWidth": 100,
            "opacity": 1,
            "paddingBottom": 6,
            "paddingHorizontal": 15,
            "paddingTop": 6,
            "paddingVertical": 10,
            "shadowColor": null,
            "shadowOffset": null,
            "shadowOpacity": null,
            "shadowRadius": null,
          }
        }
        testID="tour-already-have-account-test"
      >
        <Text
          style={
            Array [
              Object {
                "fontFamily": "Graphik-Regular",
                "letterSpacing": 0.75,
                "textAlign": "left",
              },
              undefined,
              undefined,
              Object {
                "color": "rgb(255, 255, 255)",
              },
              undefined,
              undefined,
              undefined,
              undefined,
              undefined,
              undefined,
              undefined,
              undefined,
              undefined,
              undefined,
              undefined,
              undefined,
              Object {
                "fontSize": 18,
              },
              undefined,
              undefined,
              undefined,
              undefined,
              undefined,
              undefined,
              undefined,
              undefined,
              undefined,
              undefined,
              undefined,
              undefined,
              undefined,
              undefined,
              undefined,
              undefined,
            ]
          }
        >
          I already have an account
        </Text>
      </View>
    </View>
  </RCTScrollView>
</View>
`;
